modules/task/backend.go

```diff
@@ -118,7 +93,28 @@ func (b *inMemBackend) Name() string {
 // Start implements the Module interface
 func (b *inMemBackend) Start(ready chan<- struct{}) <-chan error {
 	b.isRunning = true
-	return make(chan error)
+	errorCh := make(chan error, 2)
+	go func(buf chan []byte) {
+		for {
+			select {
+			case msg, ok := <-buf:
+				if ok {
+					errorCh <- Run(msg)
+				}
+			case <-time.After(time.Second):
+				fmt.Println("Timed out after 1 second")
+			}
+		}
+	}(b.bufQueue)
```

Comment:
why pass this as a parameter instead of just grabbing it from the closure? code clarity?